code/game/objects/items/implants/implant.dm

```diff
@@ -87,8 +87,8 @@
 	imp_in = target
 	target.implants += src
 	if(activated)
-		for(var/X in actions)
-			var/datum/action/implant_action = X
+		for(var/action in actions)
+			var/datum/action/implant_action = action
```

Comment:
this can actually be replaced with an `as anything`

Reply:
whoopsies thanks! i wasn't sure because `actions` seemed to broad to me at the time to deserve an `as anything`

Reply:
you are effectively doing an `as anything` by casting the next line, so it's identical anyways